s3auth-relay/pom.xml

```diff
@@ -174,6 +174,11 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>commons-net</groupId>
+            <artifactId>commons-net</artifactId>
+            <version>3.3</version>
```

Comment:
@krzyk I see, but when I take it out I get import errors on the commons-net classes I need. Maybe it's beacause jcabi-parent is a parent of `s3auth` but not of `s3auth-relay`?